cpu/gen/src/lib.rs

```diff
@@ -1,30 +1,30 @@
 // Licensed under the Apache-2.0 license
-use caliptra_common::memory_layout::{FMCALIAS_TBS_ORG, LDEVID_TBS_ORG};
+use caliptra_common::memory_layout::*;
 pub fn gen_memory_x(iccm_org: u32, iccm_size: u32) -> String {
     format!(
         r#"
-        ICCM_ORG  		 = 0x{:08X};
-        DCCM_ORG  		 = 0x50000000;
-        DATA_ORG  		 = 0x50004800;
-        STACK_ORG 		 = 0x5001C000;
-        ESTACK_ORG  	 = 0x5001F800;
-        NSTACK_ORG       = 0x5001FC00;
-        LDEVID_TBS_ORG   = 0x{:08X};;
-        FMCALIAS_TBS_ORG = 0x{:08X};;
-        RTALIAS_TBS_ORG  = 0x50004000;
-        
-        LDEVID_TBS_SIZE   = 1K;
-        FMCALIAS_TBS_SIZE = 1K;
-        RTALIAS_TBS_SIZE  = 1K;
-              
-        ICCM_SIZE   = 0x{:08X};
-        DCCM_SIZE   = 128K;
-        DATA_SIZE   = 94K;
-        STACK_SIZE  = 14K;
-        ESTACK_SIZE = 1K;
-        NSTACK_SIZE = 1K;
-        
-        
+        ICCM_ORG  		 = 0x{iccm_org:08X};
```

Comment:
Nit: any reason this isn't capitalized like the others? Same for `iccm_size` below.

Reply:
Because it's passed as a parameter to this function instead of a constant. I'm not sure why, that's just how it was before. Happy to change this in a follow-up PR.

Reply:
Ah I missed that.

Reply:
Leave it as-is to follow the rust style rules; constants should be upper case, variables lower-case.

Reply:
Oh; maybe you mean you can get rid of the function parameters and replace them with constants...